calling-apis/managing-asynchronous-calls.md

```diff
@@ -309,11 +334,17 @@ Which of the following most accurately describes where we print the results from
 ### !end-challenge
 <!-- prettier-ignore-end -->
 
-This solution successfully uses the results of one API call to make another API call, but it has a few drawbacks:
+Raffy chose to create a new function `getLocationFromQuery` to manage calling the existing functions `findLatitudeAndLongitude` and `findLocation`. He could have called the second method directly from inside the first with `.then` chaining, but doing so would have linked these two functions together, making it harder to use one without the other.
+
+To use `findLatitudeAndLongitude` from another function and pass on its `response` data to another network call, we needed to make a couple updates in `findLatitudeAndLongitude`: 
+- `return` the result of the `axios` call which returns the promise chain the axios call creates.
+- Inside the `.then` block, we unpack the `latitude` and `longitude` from the `response` object and return just those values we want in a new Object. This allows us to pass these values to a new `.then` block wherever we invoke the function.
```

Comment:
```suggestion
- Inside the `.then` block, we unpack the `latitude` and `longitude` from the `response` object and return just those values we want in a new Object. This allows us to pass these values to a new `.then` block wherever we invoke `findLatitudeAndLongitude`.
```

How do you feel about replacing "the function" with the name itself to be extra clear. I feel like this lesson could be challenging to wrap your mind around if it's one's first time working with async code.